composer.json

```diff
@@ -6,11 +6,14 @@
     "license": "MIT",
     "require": {
         "cakephp/console": "^4.0",
-        "rector/rector": "^0.14.5"
+        "nette/utils": "^3.2",
```

Comment:
this is needed as because the `Nette\Utils\Strings` that used in the rules are not scoped